kythe/cxx/verifier/BUILD

```diff
@@ -79,6 +79,27 @@ cc_test(
     ],
 )
 
+cc_library(
+    name = "souffle_entrystream_support",
+    srcs = [
+        "souffle_entrysteam_support.cc",
+    ],
+    deps = [
+        "//kythe/proto:common_cc_proto",
+        "//kythe/proto:storage_cc_proto",
+        "@com_google_protobuf//:protobuf",
+        "@souffle",
+    ],
```

Comment:
I *think* you may need `alwayslink = True` here as the binary isn't directly using any of the symbols from this library and they may end up getting stripped as a result.